### ISSUE TITLE
Show qr codes for the private keys

### ISSUE THREAD

**Issue by Xpl0itU** [turtlecoin/turtlecoin-mobile-wallet#88]

**Comment by SoreGums** [comment]
![image](https://user-images.githubusercontent.com/317/61960851-729c6080-b01a-11e9-8ba7-8a6e6317a3df.png)
> Wouldn’t it be better to encode a QR code using the users address and not the private keys? (re: #88 tonchan)

so which is it?

1. show a qr for the users address (already done) 
2. sow a qr for private keys

:)

**Comment by ExtraHash** [comment]
It can be useful if you want to transfer from one device to another, but there's not much screen real-estate on the phone to put two pretty big qr codes

**Comment by zpalmtree** [comment]
Currently there's nothing allowing importing from QR codes, so what's the use case?

**Comment by SoreGums** [comment]
if this was done.
probably makes sense to put all the chars into the same qr code
`send=&view=`
that's simple enough for any other program to decode or even a human to copy-paste out manually if needed.

current screen shows the text, add a link/text "show qr"
then on another device where it shows the text fields have a link/text "import via qr"

using the above idea, loop is closed

as for a practical usecase:

1. setup wallet fast on other device
2. backup keys by photocopying screen - lol
3. keep an image of keys instead of text.

seems like a lot of effort for little reward.

**Comment by zpalmtree** [comment]
I agree.. if another wallet wanted to use that method, I would support it, but I don't think I want to allow importing in this way in TonChan - so no use displaying it until someone else needs the feature.

**Comment by Xpl0itU** [comment]
I found that I need this feature when I wanted to import my wallet to another device, that's why I think it would be useful for me and others wanting to do this easily

**Comment by ExtraHash** [comment]
even though no wallet directly supports it, you can use a third party qr code reader

**Comment by Xpl0itU** [comment]
That's true
